packages/devtools_app/integration_test/test/live_connection/network_screen_test.dart

```diff
@@ -57,6 +57,16 @@ void main() {
     _expectInRequestTable('DELETE');
     await helper.clear();
 
+    // Instruct the app to make a GET request via the 'http' package.
+    await helper.triggerRequest('packageHttp/get/');
+    _expectInRequestTable('GET');
```

Comment:
this expect statement could be true because of the get request on line 41. We need to ensure we can uniquely identify the requests in the table, either with some key or by checking findsNWidgets instead which should increase for each request of the same method.

Reply:
Ah okay. Just to be certain, can we add an expect statement at the end of the helper.clear() method that verifies screenControllers.lookup<NetworkController>().requests.value is empty?

Reply:
Oh, excellent, done!